app/models/like.rb

```diff
@@ -0,0 +1,13 @@
+class Like < ApplicationRecord
+  has_many :likes, foreign_key: 'posts_id'
+  has_many :comments, foreign_key: 'posts_id'
+  belongs_to :author, class_name: 'User', dependent: :destroy, foreign_key: 'author_id'
```

Comment:
Great job @eerapheal 🤝

- [ ] According to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png), associations between models are set up correctly. Kindly make sure likes belongs to the post.
![likes](https://user-images.githubusercontent.com/63917725/215030548-04af42cc-8483-4390-bd00-2346293dccdd.PNG)